internal/component/database_observability/postgres/collector/query_samples.go

```diff
@@ -54,8 +53,7 @@ const selectPgStatActivity = `
 			(
 				coalesce(TRIM(s.query), '') != '' AND s.query_start IS NOT NULL AND
 				(
-					s.state != 'idle' OR
-					(s.state = 'idle' AND s.state_change > $1)
+					s.state != 'idle' OR					
```

Comment:
```suggestion
					s.state != 'idle'
```
Can also remove the parenthesis